index.html

```diff
@@ -177,6 +177,54 @@ <h3 class="center">Our work reflected in numbers</h3>
   </div>
 </div>
 
+<div class="section teal">
+  <div class="container row">
+    <h3 class="center">Partners and Friends of FCCVienna</h3>
+    <p class="center">If you enjoyed taking part in one of our meetings, and you feel like connecting to other people, these partners are also organizing amazing groups.</p>
```

Comment:
ReactiveConf is not a meetup group, so how about we change to instead say:
"... these partners are definitely worth checking out."